src/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  padding-top: 12px;
+}
```

Comment:
?

Reply:
I had to apply it. Otherwise, there was a small distance to the top and the test was not passed...